src/core/mac/sub_mac.cpp

```diff
@@ -276,6 +276,52 @@ void SubMac::CslSample(void)
 }
 #endif // OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE
 
+#if OPENTHREAD_CONFIG_MAC_CSL_DEBUG_ENABLE
+void SubMac::LogReceived(RxFrame *aFrame)
+{
+    Address  dst;
+    char     logString[64];
+    int32_t  drift;
+    uint32_t sampleTime, ahead, after;
+
+    IgnoreError(aFrame->GetDstAddr(dst));
+
+    VerifyOrExit((dst.GetType() == Address::kTypeShort && dst.GetShort() == GetShortAddress()) ||
+                 (dst.GetType() == Address::kTypeExtended && dst.GetExtended() == GetExtAddress()));
+
+    LogWarn("Received frame in state (SubMac %s, CSL %s), timestamp %lu", StateToString(mState),
```

Comment:
It seems strange to use LogWarn here for debug messages? Nothing to warn about (yet).

Reply:
I unintentionally pushed the changes that I was using for my testing. Fixed now.